### PR TITLE
Feature: Edit Food Page

### PR DESCRIPTION
**Pull request by JanaS87**
- created 2 form components
- added them in the food detail page
- fixed button-bug
- didn't needed the second page or a new folder
- styling not done

### CODE REVIEW

File: pages/foods/[id].js [hunk 1]
```diff
@@ -38,8 +40,12 @@ export default function FoodDetailsPage({
     cat.food.dislikes.includes(id)
   );
 
-  function handleEdit() {
-    setIsEditing(true);
+  function handleEditGood() {
```
Comment (Faibs7) [inline pages/foods/[id].js:43]: Here, for example, you could turn the "handleEditGood" into a "handleEditGoodFood". That would perhaps be a bit more specific and you would know exactly what it belongs to. If you had to take over the changes, then of course you would have to change all the places where it is used.

File: pages/foods/[id].js [hunk 2]
```diff
@@ -38,8 +40,12 @@ export default function FoodDetailsPage({
...
+    setIsEditingGood(true);
+  }
+
+  function handleEditBad() {
```
Comment (Faibs7) [inline pages/foods/[id].js:47]: here exactly the same only with "bad

**Comment by Faibs7** [comment]
Hey Jana, your code seems very understandable to me and your naming is great. However, I would like to make a few small changes regarding the naming. In my opinion, these are not a must, but they would make it even easier to understand right away. The names that refer to the editing of food or cat could be more detailed with a food/cat at the end. As I said, you have done this almost everywhere. I just don't think it's exact with your Food. Keep me posted on whether I'm right and whether it was a good idea.

File: pages/foods/[id].js (hunk 1 above)
Comment (JanaS87) [reply pages/foods/[id].js]: Thanks for your suggestions :) 

I have changed the naming of the function, I hope it's now better understandable :-)

**Pull request merged**